easybuild/easyconfigs/p/ParallelIO/ParallelIO-2.2.2a-intel-2017a.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'CMakeMake'
+
+name = 'ParallelIO'
+version = '2.2.2a'
+
+homepage = 'https://github.com/NCAR/ParallelIO'
+description = """A high-level Parallel I/O Library for structured grid applications"""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+toolchainopts = {'usempi': True}
+
+# underscore = "${version//./_}"           bash substition like this 
+# sources = ['pio%(underscore)s.tar.gz']    in an easyconfig file ?
+sources = ['pio2_2_2a.tar.gz']
+source_urls = ['https://github.com/NCAR/ParallelIO/archive/']
+checksums = ['052a03e8e3b5ac244dc53b397a9920af3066efffa216ea7f8e5e4ac73a8b8033']
+
+separate_build_dir = True
+
+dependencies = [
+  ('PnetCDF', '1.8.1', '', ('intel', '2017a')),
```

Comment:
@mhros Could you explain your thinking here? These dependencies are using the same toolchain as the parent, so I'm not clear why you've specified the suffix here?

You should just be able to use `('PnetCDF', '1.8.1'),` and `('netCDF-Fortran', '4.4.4'),`.